ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/SingleEntityVulnerabilitiesTable.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { Button, ButtonVariant } from '@patternfly/react-core';
+import { TableComposable, Thead, Tr, Th, Tbody, Td } from '@patternfly/react-table';
+import { CheckCircleIcon, ExclamationCircleIcon } from '@patternfly/react-icons';
+import { SVGIconProps } from '@patternfly/react-icons/dist/js/createIcon';
+
+import LinkShim from 'Components/PatternFly/LinkShim';
+import SeverityIcons from 'Components/PatternFly/SeverityIcons';
+import { vulnerabilitySeverityLabels } from 'messages/common';
+import { getDistanceStrictAsPhrase } from 'utils/dateUtils';
+import { ImageVulnerabilitiesResponse } from '../hooks/useImageVulnerabilities';
+import { getEntityPagePath } from '../searchUtils';
+
+export type SingleEntityVulnerabilitiesTableProps = {
+    imageVulnerabilities: ImageVulnerabilitiesResponse['image']['imageVulnerabilities'];
+};
+
+function SingleEntityVulnerabilitiesTable({
+    imageVulnerabilities,
+}: SingleEntityVulnerabilitiesTableProps) {
+    return (
+        <TableComposable>
+            <Thead>
+                <Tr>
+                    <Th>CVE</Th>
+                    <Th>Severity</Th>
+                    <Th>CVE Status</Th>
```

Comment:
i think technically PF conventions would have this be `CVE status` (https://www.patternfly.org/v4/ux-writing/capitalization)

Reply:
Oh good catch, I'll make this change now in the next PR I'm working since CI is happy with this one as-is.